endpoints/openrtb2/auction.go

```diff
@@ -218,10 +256,94 @@ func (deps *endpointDeps) validateImpExt(ext openrtb.RawJSON, impIndex int) erro
 			if err := deps.paramsValidator.Validate(bidderName, ext); err != nil {
 				return fmt.Errorf("request.imp[%d].ext.%s failed validation.\n%v", impIndex, bidder, err)
 			}
-		} else {
+		} else if bidder != "prebid" {
```

Comment:
Not at the moment, no. `isValid` is true if a string is a known `BidderName`. "prebid" is not a bidder, `isValid && bidder == "prebid"` would never be true.

Maybe you meant `isValid || bidder == "prebid"`? But that wouldn't work either because the `deps.paramsValidator.Validate` call would fail, since there's no JSON-schema for it.

Adding a JSON-schema would add it to the `/bidders/params` endpoint--which isn't good either, because callers rely on that API to build a list of demand sources.

So... I know we have to special-case "prebid" _somewhere_. I'm not sure this is the best place for it... but let me know if you can think of something cleaner.

Reply:
ah ok I see. nevermind, let's leave it as is.